src/ebmc/ebmc_base.cpp

```diff
@@ -126,11 +126,11 @@ int ebmc_baset::finish_bmc(prop_conv_solvert &solver) {
       continue;
     
     status() << "Checking " << property.name << eom;
-    
-    or_exprt or_expr;
-    
+
+    or_exprt or_expr({});
+
     for(auto l : property.timeframe_literals)
-      or_expr.operands().push_back(literal_exprt(!l));
+      or_expr.add_to_operands(literal_exprt(!l));
```

Comment:
Can we instead construct exprt::operandst here, `reserve` the right amount of space, and only thereafter create the `or_exprt`?

Reply:
sure, done